website/docs/docs/dbt-versions/release-notes/02-Nov-2023/repo-caching-rn.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "New: Support for Git repository caching"
+description: "November 2023: New option in dbt Cloud to enable Git repository caching for your job runs."
+sidebar_label: "New: Support for Git repository caching"
+sidebar_position: 07
+tags: [Nov-2023]
+date: 2023-11-29
+---
+
+Now available in dbt Cloud is a new option to enable Git repository caching for your job runs. When enabled, it tells dbt Cloud to cache your dbt project's Git repository and to use the cached copy instead if there's an outage with the Git provider. This feature helps improve the reliability and stability of your job runs. 
```

Comment:
Probably should call out the explicit enterprise plan here.

Reply:
good flag!